Cpp/odin-fg/native.jwt.cpp

```diff
@@ -69,8 +69,12 @@ const fg::frame::builtin odin::lib::jwt_payload =
         [](fg::frame &stack,
            fg::json::const_iterator pos,
            fg::json::const_iterator end) {
+            fostlib::string app_id= "";
+            if (pos!=end) {
+                app_id = stack.resolve_string(stack.argument("app_id", pos, end));
```

Comment:
I don't think the app_id should be the first argument. I think the JWT itself should be first, and then the app ID could follow.